app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_type/AddBicycleBarrierType.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BLIND
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CAR
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.PEDESTRIAN
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.WHEELCHAIR
+
+class AddBicycleBarrierType : OsmFilterQuestType<BicycleBarrierType>() {
+
+    override val elementFilter = """
+        nodes with barrier=cycle_barrier and !cycle_barrier
+    """
+    override val commitMessage = "Add specific cycle barrier type"
+    override val wikiLink = "Key:cycle_barrier"
+    override val icon = R.drawable.ic_quest_barrier // TODO
```

Comment:
@westnordost Do you have an idea or energy for making an icon?

Reply:
Same as bollard but with a bike?

Or a front-view of a bicycle barrier like "|==|"?

Reply:
I added bicycle_barrier.svg

I was lazy and just used barrier.svg

I wanted to do something like 🚳 but it did not look good.